PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/GenomeMap.java

```diff
@@ -14,13 +14,12 @@
  * This is necessary to minimize the memory consumption of storing the genome data in the graph.
  * </p>
  * <p>
- * Unless explicitly necessary, the programmer should use the IDs rather than the names.
- * This helps reduce memory usage when storing genomes.
+ * Unless explicitly necessary, the programmer should use the IDs rather than the names. This helps
+ * reduce memory usage when storing genomes.
  * </p>
  * <p>
- * This class implements the <i>Singleton design pattern</i>.
- * It can only be instantiated once.
- * This is useful because the mappings should be accessible everywhere.
+ * This class implements the <i>Singleton design pattern</i>. It can only be instantiated once. This
+ * is useful because the mappings should be accessible everywhere.
```

Comment:
Please undo these changes. It is convention to put each sentence on a new line in JavaDoc. (helps with maintainability since 1 sentence == 1 line). You should be able to set up your auto formatting to honour existing (manual) line breaks.